src/services/rateSource/CurrencyBeaconService.ts

```diff
@@ -34,3 +35,7 @@ export class CurrencyBeaconService extends RateSourceService {
     };
   }
 }
+
+export default new CurrencyBeaconService(
+  process.env.CURRENCY_BEACON_API_KEY ?? '',
```

Comment:
I think operator OR(||) fits here better since it's more general

Here is[ the quote ](https://dev.to/findniya/the-or-operator-vs-nullish-coalescing--agp#:~:text=One%20of%20the%20key%20differences,and%20undefined%20to%20be%20nullish.)about the difference between them

> One of the key differences between these two operators is how they handle falsy values. The OR operator considers any falsy value (e.g. 0, '', false) to be equivalent to false, while the nullish coalescing operator only considers null and undefined to be nullish.

Reply:
That's debatable
The explanation above is exactly a reason why I used it
Can a key be 0? Potentially yes

Reply:
makes sense)